classes/components/listPanels/PKPContributorsListPanel.inc.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @file classes/components/listPanels/PKPAnnouncementsListPanel.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPAnnouncementsListPanel
+ * @ingroup classes_components_list
+ *
+ * @brief A ListPanel component for viewing and editing email templates
```

Comment:
Update docblock

Reply:
Chenged to `@brief A ListPanel component for viewing and editing contributors`